src/Bundle/CoreBundle/View/Types/Factories/ViewConfigurationFactoryInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace UniteCMS\CoreBundle\View\Types\Factories;
+
+use UniteCMS\CoreBundle\Entity\Fieldable;
+use UniteCMS\CoreBundle\Field\FieldTypeManager;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+interface ViewConfigurationFactoryInterface
+{
+    /**
+     * @param Fieldable $fieldable
+     * @param FieldTypeManager $fieldTypeManager
+     * @return ConfigurationInterface
+     */
+    public function create(Fieldable $fieldable, FieldTypeManager $fieldTypeManager): ConfigurationInterface;
```

Comment:
`FieldTypeManager $fieldTypeManager` can be removed here and added as a dependency to each factory.

Reply:
I've made this change.